index.js

```diff
@@ -78,12 +78,27 @@ function dockerPostgres(dockerOpts) {
     // set the timeout of the hook long enough for the container to start
     this.timeout(10000);
     return new Promise(function (resolve, reject) {
-        waitForDockerContainerToBeReady(dockerOpts, passError(reject, function () {
-            state.containerReference.getConString(databaseName, passError(reject, function (conString) {
+        if (process.env.GITLAB_CI) {
```

Comment:
Can this be generalized to check for the `POSTGRES_*` variables instead? That would mean that it would work with travis as well.